xdsl/tools/command_line_tool.py

```diff
@@ -572,6 +572,11 @@ def get_replace_incompatible_fpga():
 
         return replace_incompatible_fpga.ReplaceIncompatibleFPGA
 
+    def get_stencil_inlining():
+        from xdsl.transforms import stencil_inlining
```

Comment:
Is there a cyclic import? Why not on top?

Reply:
We have a lazy import interface that avoids the 1+s import of all the files in xDSL by default when invoking `xdsl-opt`

Reply:
(For completeness)
We avoid those 1+s imports because all decorators are reapplied on import, and this just makes xDSL startup time explode